docs/content/theme/_5_writing-web-component-styles.md

```diff
@@ -415,14 +167,12 @@ You can optionally customize this set by passing in a list of just the themes yo
     }
     ```
 
-  * Naming conventions.. BEM.. @TODO
+  * Naming conventions follow the same approach as outlined on the [PatternFly Global Variables documentation.](https://www.patternfly.org/v4/documentation/overview/global-css-variables).
 
-3. Avoid use of !Important if possible
-    *   ShadyCSS adds !important to styles, which means we generally should not add !important to our stylesheets.
-    *   @TODO is this still the case? pfe-cta uses important
+3. Avoid use of `!important` if possible
```

Comment:
A good caveat might be except for slotted content where it might be necessary to override browser styles.